examples/apps/color_finder.py

```diff
@@ -239,7 +265,7 @@ def fill_in_gaps_of_pixel_matrix(self):
 
     def get_low_res_view(self):
         ''' Returns a low-resolution version of Cozmo's camera view.'''
-        image = self.robot.world.latest_image.raw_image
+        image = gray_world(self.robot.world.latest_image.raw_image)
         return image.resize((self.downsize_width, self.downsize_height), resample = Image.LANCZOS)
```

Comment:
I suspect that resize -> gray would be faster, and I doubt there would be a noticeable difference in color, but not a huge deal as long as it runs ok.